core/authentication_api.php

```diff
@@ -342,6 +342,13 @@ function auth_prepare_username( $p_username ) {
 			if( !auth_http_is_logout_pending() ) {
 				if( isset( $_SERVER['PHP_AUTH_USER'] ) ) {
 					$t_username = $_SERVER['PHP_AUTH_USER'];
+				} else if( isset( $_SERVER['REMOTE_USER'] ) ) {
```

Comment:
The processing of `REMOTE_USER` is currently done under `BASIC_AUTH` (see above).  Is there documentation for this and a reason to classify it under `HTTP_AUTH` vs. `BASIC_AUTH`?